processor/transformprocessor/README.md

```diff
@@ -307,11 +255,12 @@ metric_statements:
     - limit(datapoint.attributes, 100, ["host.name"])
 ```
 
-In this configuration, while the `datapoint` context appears suitable for parsing both `metric` 
-and `datapoint` data, it does not fulfill the function requirement since `convert_sum_to_gauge` is 
-a specific `metric` function. Conversely, the `metric` context cannot parse `datapoint`, as it 
-cannot access "lower" items in the protobuf definition. Therefore, the conflicting statements must 
-be placed into separate statement groups:
+In this configuration, the `datapoint` Path prefixed is used in the same group of statements as the `convert_sum_to_gauge`
+function. Since `convert_sum_to_gauge` can only be used with the Metric signal, not the Datapoint signal, but the list
```

Comment:
I feel like calling data points a "signal" is playing a bit fast and loose with the term, could we just say "metrics" and "data points"?

Reply:
Ya